### ISSUE TITLE
No syntax highlight for Objective-C code blocks in Markdown files

### ISSUE THREAD

**Issue by yzhao198** [microsoft/vscode#112787]
Issue Type: <b>Bug</b>

Here is a snippect of Objective-C code:

```objc
- (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event {

    // Convert the point to the target view's coordinate system.
    // The target view isn't necessarily the immediate subview
    CGPoint pointForTargetView = [self.targetView convertPoint:point fromView:self];

    if (CGRectContainsPoint(self.targetView.bounds, pointForTargetView)) {

        // The target view may have its view hierarchy,
        // so call its hitTest method to return the right hit-test view
        return [self.targetView hitTest:pointForTargetView withEvent:event];
    }

    return [super hitTest:point withEvent:event];
}
```

The syntax is highlighted fine when it is saved as an Objective-C file with `.m` extension. See the left side in the screenshot.

However, when it is saved as a code block in a Markdown file with `md` extension, there is no syntax highlight at all. See the right side in the screenshot.

<img width="1864" alt="comparison" src="https://user-images.githubusercontent.com/15700329/102572296-48d15700-40b1-11eb-936a-d7f6e39aa430.png">

VS Code version: Code 1.52.1 (ea3859d4ba2f3e577a159bc91e3074c5d85c0523, 2020-12-16T16:30:02.420Z)
OS version: Darwin x64 19.6.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz (12 x 2600)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>metal: disabled_off<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>protected_video_decode: unavailable_off<br>rasterization: enabled<br>skia_renderer: disabled_off_ok<br>video_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 2, 2|
|Memory (System)|16.00GB (0.60GB free)|
|Process Argv|--crash-reporter-id 391c2dc0-de02-43b6-b01a-cd3d286e5ef3|
|Screen Reader|no|
|VM|0%|
</details>Extensions: none

**Comment by gjsjohnmurray** [comment]
See #102548

**Comment by yzhao198** [comment]
Hi @gjsjohnmurray thank you for pointing me to the related issue. 

It looks like that issue is about adding syntax highlighting for a language that VSCode hasn't already supported. But Objective-C should have been supported by VSCode. I checked the built-in `package.json` file [here](https://github.com/microsoft/vscode/blob/7d57a8f6f546b5e30027e7cfa87bd834eb5c7bbb/extensions/markdown-basics/package.json), and `"meta.embedded.block.objc": "objc"` is already there in the `embeddedLanguages` field. Do I still need to follow [the guide](https://github.com/mjbvz/vscode-fenced-code-block-grammar-injection-example) to create an extension to add the same `meta.embedded.block.objc`?

I also installed [this extension](https://github.com/pushqrdx/code-fences) which seems to do what [the guide](https://github.com/mjbvz/vscode-fenced-code-block-grammar-injection-example) says, but it didn't fix my problem.
